app/javascript/components/editor/FileEditor.tsx

```diff
@@ -48,58 +36,6 @@ export function FileEditor({
   files: File[]
   wrap: WrapSetting
 }): JSX.Element {
-  // const languageServerUrl: string = useMemo(() => {
```

Comment:
I've extracted this to a custom hook in order for us to not comment this code out anymore.

Reply:
Great!